scenarios/python/console/captioning/build_run_app.sh

```diff
@@ -0,0 +1,62 @@
+#!/bin/bash
+
+action=$1
+
+function test_python_installed {
+    command -v python3 &>/dev/null
+}
+
+function test_pip_installed {
+    command -v pip3 &>/dev/null
+}
+
+if [[ $action == "build" ]]; then
+    if ! test_python_installed; then
+        echo "Python is not installed. Please install Python to proceed." >&2
+        exit 1
+    fi
+
+    if ! test_pip_installed; then
+        echo "pip is not installed. Please install pip to proceed." >&2
+        exit 1
+    fi
+
+    echo "Installing azure-cognitiveservices-speech package..."
+    if pip3 install azure-cognitiveservices-speech; then
+        echo "Package installation succeeded."
+    else
+        echo "Package installation failed. Please check your pip installation." >&2
+        exit 1
+    fi
+elif [[ $action == "run" ]]; then
+    envFilePath=".env/.env.dev"
```

Comment:
Please don't use the value in ".env/.env.dev" file. Use config.json instead.
For user who download this sample from GitHub repo without using our VSC extension, they don't have the ".env/.env.dev" file. We should make the build_run_app scripts rather general for them, too.